precompile/assetsbridge/min_amount.go

```diff
@@ -207,3 +207,134 @@ func (m *GetMinBridgeOutAmountMethod) Run(
 		precompile.TypesConverter.BigInt.FromSDK(minAmount),
 	}, nil
 }
+
+// SetMinBridgeOutAmountForBitcoinChainMethodName is the name of the setMinBridgeOutAmountForBitcoinChain
+// method. It matches the name of the method in the contract ABI.
+//
+//nolint:gosec
+const SetMinBridgeOutAmountForBitcoinChainMethodName = "setMinBridgeOutAmountForBitcoinChain"
+
+// SetMinBridgeOutAmountForBitcoinChainMethod is the implementation of the setMinBridgeOutAmountForBitcoinChain
+// method.
+type SetMinBridgeOutAmountForBitcoinChainMethod struct {
+	poaKeeper    PoaKeeper
+	bridgeKeeper BridgeKeeper
+}
+
+func newSetMinBridgeOutAmountForBitcoinChainMethod(
+	poaKeeper PoaKeeper,
+	bridgeKeeper BridgeKeeper,
+) *SetMinBridgeOutAmountForBitcoinChainMethod {
+	return &SetMinBridgeOutAmountForBitcoinChainMethod{
+		poaKeeper:    poaKeeper,
+		bridgeKeeper: bridgeKeeper,
+	}
+}
+
+func (m *SetMinBridgeOutAmountForBitcoinChainMethod) MethodName() string {
+	return SetMinBridgeOutAmountForBitcoinChainMethodName
+}
+
+func (m *SetMinBridgeOutAmountForBitcoinChainMethod) MethodType() precompile.MethodType {
+	return precompile.Write
+}
+
+func (m *SetMinBridgeOutAmountForBitcoinChainMethod) RequiredGas(_ []byte) (uint64, bool) {
+	return 0, false
+}
+
+func (m *SetMinBridgeOutAmountForBitcoinChainMethod) Payable() bool {
+	return false
+}
+
+func (m *SetMinBridgeOutAmountForBitcoinChainMethod) Run(
```

Comment:
When setting the general minimum bridge-out amount we emit an event.
We could consider emitting an event here for consistency.

Reply:
See 236dcc18d37c984b05103ee7c69e62fdafee6b5d